app/components/edit-text/template.hbs

```diff
@@ -1,8 +1,8 @@
 {{#if (eq type 'title')}}
     {{#if editMode}}
-        {{input class="edit-text-title"  maxlength=140 value=layer.settings.values.sectionTitle}}
+        {{input type="text" class=layer.settings.values.alignment maxlength=140 value=layer.settings.values.sectionTitle }}
```

Comment:
Somewhere else you took the input out of helper and just used <input>. The helper is really just a wrapper, you can use tags too.